### DISCUSSION TITLE
Passthrough for node is not working for Tree component.

### DISCUSSION THREAD

**Discussion by farazshuja** [primefaces/community#2723] [PrimeVue] [answered]
As per the documentation https://primevue.org/tree/#api.tree.interfaces.TreePassThroughOptions it says that I can pass custom props to node, nodeLabel. But in actual its not setting any attribute to the DOM nodes. 

Here is the sandbox: https://codesandbox.io/p/devbox/k3fkfk

```
<Tree
      v-model:selectionKeys="selectedKey"
      :value="nodes"
      selectionMode="single"
      pt:node:data-test-id="12345"
      class="w-full md:w-[30rem]"
      @nodeSelect="onNodeSelect"
      @nodeUnselect="onNodeUnselect"
    ></Tree>
```

**Comment by zaid1523** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/c7446ea1-7e79-4bc6-b75b-771a90dd8287)
![image](https://github.com/user-attachments/assets/ab3a3cc6-2d73-4f3c-95bc-d6a3336ec25e)
use PassThrough correctly it will work.
